client/my-app/src/Components/ReadingSchedule/ReadingSchedule.tsx

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import styles from "./ReadingSchedule.module.css";
+import {
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  Button,
+  Checkbox,
+  TextField
+} from "@mui/material";
+import { DatePicker, LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { DemoContainer } from "@mui/x-date-pickers/internals/demo";
+
+interface ReadingScheduleProps {}
+
+const ReadingSchedule = (props: ReadingScheduleProps) => {
+  const [chapterAssignmentTitle, setChapterAssignmentTitle] =
+    useState<string>();
+  const [assignmentDescription, setAssignmentDescription] = useState<string>();
+  const [bookTitle, setBookTitle] = useState<string>();
+  const [assignedChapter, setAssignedChapter] = useState<string>();
+  const [date, setDate] = useState<Date | null>(null);
```

Comment:
change it to 
const [date, setDate] = useState<Dayjs | null>(null);

Reply:
Fixed